src/node_os.cc

```diff
@@ -257,7 +257,7 @@ static void GetInterfaceAddresses(const FunctionCallbackInfo<Value>& args) {
     }
 
     snprintf(mac,
-             18,
+             arraysize(mac),
```

Comment:
IMHO, if we're refactoring this it's a great opportunity to switch to `std::array<char,18>`. And use explicitly the built in `mac.size()` and `mac.data()` accesors.
Also if you can move the declaration close to it's first use, that's beneficial, to reduce the change of using the variable in a wrong way.

Reply:
Ok, Updated